.golangci.yml

```diff
@@ -124,7 +124,7 @@ linters-settings:
 
   nolintlint:
     # Enable to ensure that nolint directives are all used. Default is true.
-    allow-unused: false
+    allow-unused: true
```

Comment:
I did this because I had platform dependent lint warnings and this seems like the only wait to deal with that reasonably.